tensorrt_llm/inputs/registry.py

```
@@ -194,6 +195,30 @@ def __call__(
+        cache = get_prefix_token_cache() if prefix_cache_enabled() else None
+        use_cache = (cache is not None and isinstance(prompt, str)
+                     and len(prompt) >= cache.min_chars
+                     and "query" not in inputs
+                     and getattr(self.tokenizer, "is_fast", False)
+                     and not sampling_params.add_special_tokens
+                     and sampling_params.truncate_prompt_tokens is None)
+        if use_cache:
+            with nvtx_range_debug("tokenize prompt (prefix cache)"):
+                try:
+                    return cache.encode(self.tokenizer, prompt), None
+                except Exception:
```

Ruff (0.16.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)